nix/haskell.nix

```diff
@@ -13,33 +13,40 @@
 }:
 let
 
+  src = haskell-nix.haskellLib.cleanGit {
+      name = "cardano-prelude-src";
+      src = ../.;
+  };
+
   # This creates the Haskell package set.
   # https://input-output-hk.github.io/haskell.nix/user-guide/projects/
-  pkgSet = haskell-nix.cabalProject {
-      src = haskell-nix.haskellLib.cleanGit { src = ../.; };
-      ghc = buildPackages.haskell-nix.compiler.${compiler};
+  pkgSet = haskell-nix.cabalProject  (lib.optionalAttrs stdenv.hostPlatform.isWindows {
+  } // {
```

Comment:
What's this for? 
```
lib.optionalAttrs stdenv.hostPlatform.isWindows {}
```
is `{}` independent of what `isWindows` is. If it's false, it will be `{}`, if it's true it's also `{}`?

Reply:
yeah, this is a noop

Reply:
Vestigial remnant.  Removed.  Thanks!